src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -177,5 +191,10 @@ private void ThrowIfDisposed()
                 throw new ObjectDisposedException(null);
             }
         }
+
+        public IScaleMonitor GetMonitor()
+        {
+            return new ServiceBusScaleMonitor(_functionId, _entityType, _entityPath, _serviceBusAccount.ConnectionString, _receiver, _loggerFactory);

```

Comment:
Let's not new one of these up each time we're called. Instead, can you either create it in the ctor, or define a field for it and create once/cache in this method?